modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -213,6 +213,8 @@ public class IgniteImpl implements Ignite {
     /** Lifecycle manager. */
     private final LifecycleManager lifecycleManager;
 
+    private final ThreadPools threadPools;
```

Comment:
I'd preserve (almost)common naming patter here, meaning that I'd use Manager postfix.

Reply:
I'm not sure it makes a lot of sense naming it a manager as it's just a collection of pools, so it would be a stretch

Reply:
After thinking a bit more, I renamed the class, thanks for the suggestion.